host/src/lib.rs

```diff
@@ -12,15 +12,16 @@ use bt_hci::cmd::status::ReadRssi;
 use bt_hci::cmd::{AsyncCmd, SyncCmd};
 use bt_hci::param::{AddrKind, BdAddr};
 use bt_hci::FromHciBytesError;
-#[cfg(feature = "security")]
-use heapless::Vec;
 use rand_core::{CryptoRng, RngCore};
 
 use crate::att::AttErrorCode;
 use crate::channel_manager::ChannelStorage;
 use crate::connection_manager::ConnectionStorage;
 #[cfg(feature = "security")]
-pub use crate::security_manager::{BondInformation, IdentityResolvingKey, LongTermKey};
+pub use crate::security_manager::{IdentityResolvingKey, LongTermKey, BondInformation};
```

Comment:
I think think these should maybe be available in the trouble_host::prelude:: and trouble_host::security:: API-wise.